recipes/pysynphot/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "pysynphot" %}
+{% set version = "1.0.1" %}
+{% set sha256 = "1c301bc9780dd03062d8d0f73e771de80766a61dccff23f7524cb91adf468f5c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [py<37 or win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - numpy
+  run:
+    - python
+    - astropy
+    - {{ pin_compatible('numpy') }}
+    - beautifulsoup4
+    - six
+    - setuptools
+
+test:
+  imports:
+    - pysynphot
+
+about:
+  home: https://github.com/spacetelescope/pysynphot
+  license:  BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE.md
+  summary: 'Synthetic photometry package from Astrolib.'
+  doc_url: https://github.com/spacetelescope/pysynphot/tree/master/doc
```

Comment:
I think the doc should point to https://pysynphot.readthedocs.io/en/latest/

Reply:
Thanks for pointing this out. I'll look into how to fix this, as I'm totally new to the conda-forge workflow and this PR has sadly already merged (they're _fast_!)